packages/vue-app/template/server.js

```diff
@@ -103,7 +103,7 @@ export default async (ssrContext) => {
     __webpack_public_path__ = joinURL(ssrContext.nuxt.config.app.cdnURL, ssrContext.nuxt.config.app.assetsPath)
   }
   // Create the app definition and the instance (created for each request)
-  const { app, router<%= (store ? ', store' : '') %> } = await createApp(ssrContext, { ...ssrContext.runtimeConfig.public, ...ssrContext.runtimeConfig.private })
+const { app, router<%= (store ? ', store' : '') %> } = await createApp(ssrContext, ssrContext.runtimeConfig.private)
```

Comment:
@pi0 was this an intentional change? it looks like the server went from receiving the public and private runtimeConfig to just receiving the private one.

Reply:
https://github.com/nuxt/nuxt.js/pull/8855/files#diff-895d4548da1de2d3970ec23e8b55f69e181a9f28296b1c96504b4230ebfa4336R300

They get merged here

Reply:
If nothing as intended would you please open an issue with reproduction?